### PR TITLE
[dc-231] Sl Add swagger-ui.html support - open api symlink

### PR DESCRIPTION
**Pull request by slucasbroad**

### CODE REVIEW

File: service/src/main/java/bio/terra/catalog/config/WebConfig.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package bio.terra.catalog.config;
+
+// import bio.terra.app.logging.LoggerInterceptor;
```
Comment (lu-c) [inline service/src/main/java/bio/terra/catalog/config/WebConfig.java:3]: please remove this unused import!

**Review by petesantos — APPROVED** [review]
LGTMob

File: service/src/main/resources/api/swagger-ui.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta
+    name="description"
+    content="SwaggerIU"
```
Comment (fboulnois) [inline service/src/main/resources/api/swagger-ui.html:10]:
```suggestion
    content="Data Catalog SwaggerUI"
```

**Review by fboulnois — APPROVED** [review]
One minor suggestion:

**Pull request merged**